index.html

```diff
@@ -14,11 +14,11 @@
 
 var uncleanWords = ["Fuck","Cunt","Shit"]
 var uncleanFillers = ["no shit","fuck yeah"]
-var uncleanPhrases = ["We aren't here to fuck spiders","Fucks goin' on cunt","See you next tuesday","As useful as taste buds in an arsehole","Fucking dog cunt","Unfucking believable","Your fucked mate","Do I fuckin know you","You fuckin stingy cunt","Get the fuck out of here","Fuckin listen to me mate","You're a dickhead mate","Shits fucked","Taking the piss","Flying fuck","Fucken oath"]
+var uncleanPhrases = ["We aren't here to fuck spiders","Fucks goin' on cunt","See you next tuesday","As useful as taste buds in an arsehole","Fucking dog cunt","Unfucking believable","Your fucked mate","Do I fuckin know you","You fuckin stingy cunt","Get the fuck out of here","Fuckin listen to me mate","You're a dickhead mate","Shits fucked","Taking the piss","Flying fuck","Fucken oath","Fight me cunt","Lets get fuckin drunk!","Come get fucked up with us!"]
 var uncleanWords = ["Pisshead","Brick shit house","Cunt","Kent","Tits","Jugs","Kazza","Shindig","Yank","Seppo"]
```

Comment:
Can you remove the punctuation at the end? The ! stuff... it breaks it cos the code insert it itself. 

Even better, is make it use . say 80% of the time and the other 20% it picks either ! or ? and uppercases the next letter.

Reply:
Done